### DISCUSSION TITLE
Scraper from past bin raw

### DISCUSSION THREAD

**Discussion by maikelrichard** [iw4p/proxy-scraper#25] [Ideas]
how can i add scraper to pastbin raw format

https://pastebin.com/raw/ZspCCZEL

**Comment by iw4p** [comment]
Hello,
Sorry, I didn't understand.
You want to send and upload scraper result to Pastebin?

**Comment by maikelrichard** [comment]
not.
I would like to make a scraper using a pastbin raw link
example 

# From PastBin
class PastBinScraper(Scraper):

    def __init__(self, method):
        super().__init__(method, " https://pastebin.com/raw/ZspCCZEL ").  

for format 

IP:PORT

**Reply by iw4p** [reply]
Why not using `curl https://pastebin.com/raw/ZspCCZEL -o foo.txt`?
